--- node_modules/@theia/workspace/lib/browser/workspace-frontend-contribution.js	2019-09-04 21:09:18.662000000 -0700
+++ new.js	2019-09-04 21:36:02.015934381 -0700
@@ -409,7 +409,7 @@
                         return [4 /*yield*/, dialog.open()];
                     case 1:
                         if (!_a.sent()) return [3 /*break*/, 3];
-                        return [4 /*yield*/, this.workspaceService.close()];
+                        return [4 /*yield*/, document.location.href = "http://code.cs.wallawalla.edu/dashboard/"];
                     case 2:
                         _a.sent();
                         _a.label = 3;
